superset-frontend/src/dashboard/components/Header/HeaderActionsDropdown/index.jsx

```diff
@@ -164,11 +164,12 @@ class HeaderActionsDropdown extends React.PureComponent {
         break;
       }
       case MENU_KEYS.TOGGLE_FULLSCREEN: {
+        const hasStandalone = !!getUrlParam(URL_PARAMS.standalone);
         const url = getDashboardUrl(
           window.location.pathname,
           getActiveFilters(),
           window.location.hash,
-          getUrlParam(URL_PARAMS.standalone),
+          !hasStandalone,
```

Comment:
This comes down to taste, but I think the double bang could be done directly here (seems clear enough).

Reply:
the "negation" is needed here, the `hasStandalone` has been boolean type.

Reply:
Right, gotcha 👍